800.renames-and-merges/a.yaml

```diff
@@ -32,6 +32,8 @@
 - { setname: actiona,                  name: actionaz, ruleset: chocolatey }
 - { setname: activeio,                 name: activemq-activeio }
 - { setname: adabooch,                 name: [booch95,gnat-booch95] }
+- { setname: adapta-gtk-theme,         name: [adapta,gtk-theme-adapta] }
+- { setname: adapta-gtk-theme,         namepat: "adapta-gtk-theme.*", addflavor: true }
```

Comment:
No, this is too wide. Version suffixes (including `-legacy` and alike) should not be flavored, variants should be. That is, only `colorpack` and `wal` variants need addflavor. Better just to list them verbatim here, without regexps.